.github/release-drafter.yml

```
@@ -0,0 +1,52 @@
+      - "patch"
+      - "bug"
+      - "fix"
+  default: patch
+
+template: |
+  ## Summary
+  $CHANGES
+
+  ## Contributors
+  $CONTRIBUTORS
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)